README_EN.md

```diff
@@ -0,0 +1,141 @@
+<img src="http://opendatadpc.maps.arcgis.com/sharing/rest/content/items/5c8ef7516b5b4bb19f61037b4cd69015/data" alt="COVID-19" data-canonical-src="http://opendatadpc.maps.arcgis.com/sharing/rest/content/items/5c8ef7516b5b4bb19f61037b4cd69015/data" width="400" />
+
+# Italy's COVID-19 Data
+
+[![GitHub license](https://img.shields.io/github/license/pcm-dpc/COVID-19)](https://github.com/pcm-dpc/COVID-19/blob/master/license)
+[![GitHub commit](https://img.shields.io/github/last-commit/pcm-dpc/COVID-19)](https://img.shields.io/github/last-commit/pcm-dpc/COVID-19)
+
+[Website of the Dipartimento della Protezione Civile - Emergenza Coronavirus: la risposta nazionale](http://www.protezionecivile.it/attivita-rischi/rischio-sanitario/emergenze/coronavirus)
+
+
+On the 31 January 2020, the Council of Ministers declared a emergency of 6 month duration, in response to the health risk associated with the corona virus infection. The Head of the Civil Protection Department, Angelo Borrelli, was appointed as the coordinator of all the interventions necessary to deal with the emergency.  
+
+The main goals of the department were aimed at the rescue and assistance of the population affected by the infection, the strengthening of checks in airports and port areas, the repatriation of Italian citizens in areas deemed at risk and the repatriation of foreign citizens to their countries of origin if they were exposed to the infection.
```

Comment:
```suggestion
The main goals of the department are aimed at the rescue and assistance of the population affected by the infection, the strengthening of airports and port areas checks, the repatriation of Italian citizens in areas deemed at risk and the repatriation of foreign citizens to their countries of origin if they were exposed to the infection.
```

Reply:
If they were exposed -> In case they have been exposed

Reply:
Foreign citizens to their countries of origin if they were exposed -> foreign citizens, in case their countries of origin are exposed to risk of infection